docs/modules/ROOT/pages/how-tos/configuring/component-nudges.adoc

```diff
@@ -1 +1,81 @@
-= Defining component relationships
\ No newline at end of file
+= Defining component relationships
+
+When an application includes multiple components, their repositories might contain references to image builds of another component. For instance, an application might include multiple child components that, using a Dockerfile `FROM` clause, reference the image build of a common parent.
+
+In such instances, whenever you build a new image of the common parent component, you need to update the references to it. Specifically, you need to copy and paste the pullspec and image digest of that new image to your other source repositories. This process is tedious and error-prone.
+
+If you build an application with {ProductName} and make references between components with image digests (`sha256:...`), {ProductName} can automatically generate pull requests to update those references. To use this functionality, simply define the relationships between your components, either in the {ProductName} UI, or in your CLI.
+
+== In the UI
+
+.Procedure
+
+To define the relationships between components in an application, complete the following steps in the {ProductName} UI:
+
+. Navigate to your application. 
+. On any tab, use the *Actions* drop-down menu to select *Define component relationships*. 
+.. Alternatively, go to the *Components* tab and select the *Define component relationships*.
+. In the *Component relationships* window, select one component from the *Select a component* drop-down menu.
+. Select *Nudges* or *Is nudged by*, depending on the relationship you wish to define.
+
++
+[NOTE]
+====
+Component A _nudges_ Component B (or Component B _is nudged by_ Component A) if Component B contains a reference by image digest to a build of Component A. 
+====
+
+. Use the remaining drop-down menu to choose which other components belong to this relationship.
+. To define multiple relationships, select *Add another component relationship*.
+. Once you have defined all necessary relationships, select *Save relationships*.
+
+.Verification
+
+To verify the new relationship in the UI:
+. Go to the *Components* tab for your application. 
+. Select a component that belongs to the relationship you defined.
+. Scroll to the end of the page and select *Show related components*. 
+
+Alternatively, in the git repo for the component that nudges, push a commit. When the build for that component completes, you should see a new pull request appear for the component that is nudged. This new pull request contains the image digest for the new build of its parent image.
+
+
+.Troubleshooting
+
+* On any tab, use the *Actions* drop-down menu to select *Define component relationships*. If you do not see the relationship, try to define it again and then make sure to select *Save relationships*.
+* Ensure that the existing references in the repositories of your components are correct.
```

Comment:
It is also possible to troubleshoot the _nudging_. This will happen via a Renovate pipeline run in the tenant workspace (you will have to use the CLI to see the PLR as it will not show up in the UI).

Reply:
I'm afraid you lost be a bit here--exactly what steps would users take to troubleshoot the nudging in the CLI?

Reply:
`oc get pr` i.e. the same flow as using the `oc` CLI to get information from the workspace, just getting pipelineruns

Reply:
Ok so you use `oc` to get the PR itself. Makes sense. But then users would need to do something with that PR, too, right?  It sounded like something involving Renovate Bot?

Reply:
It might be too much detail for here. Maybe better for a SOP? They would need to get the logs for the pipeline run to see if it is working properly or if it is amiss.

Reply:
Ok so should I just leave it out for this current iteration of the doc? Something we come back to?

Reply:
Yeah, I talked myself into leaving it out.